.clang-format.chromium

```diff
@@ -0,0 +1,60 @@
+# Defines the ANGLE style for automatic reformatting.
+# https://code.google.com/p/angleproject/wiki/CodingStandard
+# See Clang docs: http://clang.llvm.org/docs/ClangFormatStyleOptions.html
+BasedOnStyle: Chromium
+
+# Allow double brackets such as std::vector<std::vector<int>>.
+Standard: Cpp11
+
+# Indent 4 spaces at a time.
+IndentWidth: 4
```

Comment:
2?

Reply:
I'm fine with 2. This one was a verbatim copy of the file from Chromium tree.

Reply:
Discussed in the community meeting, use 2 to follow Google Naming Convention.

Reply:
Changed this to 2 in .clang-format